core/ir/opnd_api.h

```diff
@@ -2225,6 +2227,13 @@ opnd_create_base_disp_arm(reg_id_t base_reg, reg_id_t index_reg,
 #endif
 
 #ifdef AARCH64
+DR_API
+/**
+ * Returns the left shift amount from \p size.
+ */
+uint
+opnd_size_to_shift_amount(opnd_size_t size);
```

Comment:
New public interfaces like this should be added to the changelist in api/docs/release.dox.

Reply:
Doh! Corrective patch will appear shortly.

Reply:
Addressed in this PR: https://github.com/DynamoRIO/dynamorio/pull/5855